src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/toolkit/util/VisibleMemberTable.java

```diff
@@ -258,7 +258,7 @@ public ExecutableElement getSimplyOverriddenMethod(ExecutableElement e) {
 
         OverridingMethodInfo found = overriddenMethodTable.get(e);
         if (found != null && found.simpleOverride) {
-            return found.overrider;
+            return found.overridden;
```

Comment:
changing from `overrider` to `overridden` seems like a pretty big change, and not just a terminology cleanup.

Is this related to the change you described in the introduction?